### PR TITLE
Add option to configure startup delay

### PR DESCRIPTION
**Pull request by and3rson**
This PR adds `-d/--delay` option to customize initial startup delay. My systems boots really fast and I usually start typing as soon as my terminal autostarts, so I'm risking to keep some keys pressed while `evremap` is waiting for me to release all keys.

This is my first time writing in Rust, so please let me know if I'm doing anything wrong!

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -66,14 +70,18 @@ fn main() -> Result<()> {
...
                 config_file.display()
             ))?;
 
             log::error!("Short delay: release any keys now!");
-            std::thread::sleep(Duration::new(2, 0));
+            let float_delay_result = FloatDuration::seconds(delay).to_std();
+            match float_delay_result {
+                Ok(v) => std::thread::sleep(v),
+                Err(e) => log::error!("Invalid delay: {e:?}"),
+            }
```
Comment (wez) [inline src/main.rs:84]: We can avoid the external dependency by using:

https://doc.rust-lang.org/std/time/struct.Duration.html#method.from_secs_f64

```suggestion
            std::thread::sleep(Duration::from_secs_f64(delay));
```

File: src/main.rs [hunk 2]
```diff
@@ -1,8 +1,8 @@
 use crate::mapping::*;
 use crate::remapper::*;
 use anyhow::{Context, Result};
+use float_duration::FloatDuration;
```
Comment (wez) [inline src/main.rs:4]: As mentioned below, we can eliminate this

```suggestion
```

File: Cargo.toml [hunk 3]
```diff
@@ -14,3 +14,4 @@ serde = { version="1.0", features=["derive"]}
 thiserror = "1.0"
 toml = "0.5"
 libc = "0.2"
+float_duration = "0.3.3"
```
Comment (wez) [inline Cargo.toml:17]: As mentioned below, we can eliminate this

```suggestion
```

**Review by wez — CHANGES_REQUESTED** [review]
Thanks!  I think we can simplify and remove the additional dependency

**Comment by and3rson** [comment]
Done! I've pushed the suggested fixes.

**Comment by wez** [comment]
Thanks!

**Pull request merged**